### ISSUE TITLE
[Accessibility]A11y_AzureDataStudio_Jobs_ScreenReader: NVDA is not announcing status while page is loading state.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#6046]
[Check out Keros!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags: #A11yMAS; #A11yTCS; #MAS1.3.1; #A11y_AzureDataStudio; #Win32; #DesktopApp

**Environment Details:**
Application name: Azure Data Studio
Version: 1.8.0-insider (user setup)
Commit: 2ca39b5
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.17763
NVDA Version: 2019.1

**Additional Details:**
Tools Name: NVDA
MAS Number :MAS1.3.1

**Steps to Reproduce :**
1. Install the app Azure Data Studio.
2. Enable the SQL server agent extension and Select database from the left panel navigate to the SQL agent tab.
3. Start NVDA.
4. Observe that while page is loading state ,screen reader doesn't announce "Page is still loading".

**Actual :**
When page is in loading state ,screen reader is not announcing "Page is still loading ".It only announces the information which present on that page.

**Expected:**
Screen reader should announce "Page is still loading " when page is loading.

**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Screen reader only user won't understand that page is loading and may misunderstood that page does not contain any content.

**MAS Reference:
MAS1.3.1-**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment for Reference:**
[6046_[Accessibility]A11y_AzureDataStudio_Jobs_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3295341/6046_.Accessibility.A11y_AzureDataStudio_Jobs_ScreenReader.pptx)

Does this issue occur when all extensions are disabled?: Yes
